topics/introduction/tutorials/galaxy-intro-101/tutorial.md

```diff
@@ -529,7 +531,7 @@ We can examine the workflow in Galaxy's workflow editor. Here you can view/chang
 > We could **validate** our newly built workflow by running it on the same input datasets that we used at the start of this tutorial, in order to make sure we do obtain the same results.
 {: .comment}
 
-## Run workflow on different data
+## Running workflow on different data
```

Comment:
renaming this caused a break in another tutorial, it's the linting message you're seeing. So glad we have good linting, could you please update that to `running-workflow-on-different-data`


* At ./_site/training-material/topics/variant-analysis/tutorials/dunovo/tutorial.html:1441:

  internally linking to ../../../introduction/tutorials/galaxy-intro-101/tutorial.html#run-workflow-on-different-data; the file exists, but the hash 'run-workflow-on-different-data' does not

Reply:
Oh ... great. I'll rename ... I'll be more careful in my future edits!